tests/json/echo.c3

```diff
@@ -21,4 +21,4 @@ function main : () -> int = {
 
 /// @COMPILE
 /// @RUN; EXPECT OUT
-/// {"children": [[]{"children": [[]{"name": "ESCAPED_STRING", "value": "std/format.c3"}[]], "name": "require_once"}[]], "name": "program"}
+/// {"children": [[]{"children": [[]{"name": "ESCAPED_STRING", "value": ""std/format.c3""}[]], "name": "require_once"}[]], "name": "program"}
```

Comment:
Is this a debugging utility or an actual JSON output? Shouldn't we also print the `\"`s?

Reply:
Actual JSON output - I've fixed this now and strings are escaped properly